terraform/variables.tf

```diff
@@ -71,25 +71,6 @@ variable "fullnode_seed" {
   description = "Default seed for fullnode network"
 }
 
-variable "fullnode_distribution" {
-  type        = list(number)
-  default     = [1, 0, 0, 0]
-  description = "List of number of fullnodes on each validator"
-}
-
-# This is to generate a list of fullnode with validator index to indicate
-# which validator they should be connected to
-locals {
```

Comment:
Thanks for cleaning this up.  TIL we do the fullnode-validator paring in fullnode.tf